google_chat_ros/Button[] buttons
string text_paragraph # Display a text paragraph in this widget
google_chat_ros/Image image # Display an image in this widget
google_chat_ros/KeyValue key_value # Display a key value item in this widget
